sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -144,9 +150,7 @@ class FileCatalog(Base):
     end_date = Column(DateTime, nullable=False)
     version = Column(String(8), nullable=False)
     extension = Column(EXTENSIONS, nullable=False)
-    status_tracking_id = Column(
```

Comment:
I guess this means that there is no link between the FileCatalog and StatusTracking anymore? I _think_ that makes sense, but just wanted to verify we don't need that for any other reason.